docs/usage/cli.md

```diff
@@ -68,6 +68,14 @@ Compile the entire `src` directory and output it to the one concatenated file.
 $ babel src --out-file script-compiled.js
 ```
 
+### Ignore files
+
+Ignore spec and test files
+
+```sh
+$ babel src --out-dir lib --ignore spec.js,test.js
```

Comment:
Thanks, although it does seem weird in to me in this example because the tests are usually in a different folder than `src` so it wouldn't need to be ignored anyway?